cmd/clustersynchro-manager/main.go

```diff
@@ -1,34 +1,31 @@
 package main
 
 import (
-	"fmt"
-	"math/rand"
 	"os"
 	"time"
 
+	"github.com/spf13/cobra"
+	"k8s.io/component-base/cli"
+	"math/rand"
+
 	apiserver "k8s.io/apiserver/pkg/server"
 	"k8s.io/component-base/logs"
 
 	"github.com/clusterpedia-io/clusterpedia/cmd/clustersynchro-manager/app"
 )
 
 func main() {
-	if err := runClusterSynchroManagerCommand(); err != nil {
-		fmt.Fprintf(os.Stderr, "%v\n", err)
-		os.Exit(1)
-	}
+	command := runClusterSynchroManagerCommand()
+	code := cli.Run(command)
+	os.Exit(code)
 }
 
-func runClusterSynchroManagerCommand() error {
+func runClusterSynchroManagerCommand() *cobra.Command {
```

Comment:
As we used `k8s.io/component-base/cli` , we don't need `runClusterSynchroManagerCommand()` anymore.